hack/windows/Install-OVS.ps1

```diff
@@ -44,48 +31,79 @@ $ErrorActionPreference = "Stop"
 $OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-3.0.5-antrea.1-win64.zip"
 # Use a SHA256 hash to ensure that the downloaded archive is correct.
 $OVSPublishedHash = '813a0c32067f40ce4aca9ceb7cd745a120e26906e9266d13cc8bf75b147bb6a5'
+# $MininalVCRedistVersion is the minimal version required by the provided Windows OVS binary. If a higher
+# version of VC redistributable file exists on the Windows host, we can skip the installation.
+$MininalVCRedistVersion="14.12.25810"
+
 $WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
-$OVSDownloadDir = $WorkDir
+$InstallLog = "$WorkDir\install_ovs.log"
 $PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$OVSZip=""
 
-if (!$LocalFile) {
-    $OVSZip = "$OVSDownloadDir\ovs-win64.zip"
-} else {
-    $OVSZip = $LocalFile
-    $DownloadDir = Split-Path -Path $LocalFile
-}
+function PrepareOVSLocalFiles() {
+    if ($LocalFile -ne "") {
+        validateLocalOVSFiles
+        $OVSDownloadURL = ""
+        $OVSPublishedHash = ""
+    } else {
+        $OVSZip = "$WorkDir\ovs-win64.zip"
+        if ($DownloadURL -ne "" -and $DownloadURL -ne "$OVSDownloadURL") {
+            $OVSDownloadURL = $DownloadURL
+            $OVSPublishedHash = ""
+        }
+    }
 
-if ($DownloadDir -ne "") {
-    $OVSDownloadDir = $DownloadDir
+    if (($OVSZip -ne "")) {
+        if (Test-Path -Path $OVSInstallDir) {
+            Log "$OVSInstallDir already exists, exit OVS installation."
+            exit 1
```

Comment:
Is this expected to exit 1 when skip OVS installation?

Reply:
yes